### DISCUSSION TITLE
How to change default browser in laragon ?

### DISCUSSION THREAD

**Discussion by mehdyfattahi** [leokhoa/laragon#560] [Q&A]
Hi
How to change default browser in laragon ?

**Comment by Pen-y-Fan** [comment]
Laragon doesn't set your default browser, it's a Windows setting. Google search 'How do I set my default browser in Windows 11.

**Comment by mehdyfattahi** [comment]
No . It is not
All virtual servers & localhosts can choose their default browser. No default Windows settings...

**Reply by Pen-y-Fan** [reply]
Sorry, are we discussing the same thing? A browser is Google Chrome, Microsoft Edge, Firefox etc. Laragon doesn't have the option to override the default browser.

> All virtual servers & localhosts can choose their default browser.

Can you give me more information? What is the source of this?
